### ISSUE TITLE
Using mouse click 3 (mouse wheel click) on a notice gives error 404

### ISSUE THREAD

**Issue by zhinu** [dis-moi/extension#896]
**Describe the bug**
When you click to view the detail of a notice with a mouse click 3, then you go to a page that doesn't exist (on firefox, haven't yet tried with chrom*).
the link you go to is {originwebsiteurl}/notices/details/{xxx}

**Expected behavior**
Should go to detail, or no behaviour at all

*
**Desktop (please complete the following information):**]
 - Browser : firefox

**Extension**
- Version : latest stable
- Installed from firefox store

**Comment by lutangar** [comment]
Related to #755

**Comment by JalilArfaoui** [comment]
Yes, but not exactly the same … (click link inside message vs click message detail from messages list)

**Comment by JalilArfaoui** [comment]
We have to see if we can prevent the link to open in a new tab … since it has no meaning

**Comment by shivang-tripati** [comment]
can i work on this
